db/types/base.py

```diff
@@ -178,6 +178,7 @@ def _add_to_db_type_hintsets(db_types, hints):
         PostgresType.NUMERIC,
         PostgresType.REAL,
         PostgresType.MONEY,
+        MathesarCustomType.MATHESAR_MONEY,
```

Comment:
This will be correct after merging my current branch, but it's currently a composite type.  I suppose it's comparable either way.

Reply:
@mathemancer If you're referring to the multicurrency type, we should not have it in any APIs until there's a design for it, see #656